Loop/Managers/Alerts/AlertManager.swift

```diff
@@ -174,21 +180,22 @@ public final class AlertManager {
         var scheduledNotifications: [StoredLoopNotRunningNotification] = []
 
         for (minutes, isCritical) in [(20.0, false), (40.0, false), (60.0, true), (120.0, true)] {
-            let failureInterval = lastLoopDate.addingTimeInterval(.minutes(minutes)).timeIntervalSinceNow
-            guard failureInterval >= 0 else { break }
+            let warningInterval = TimeInterval(minutes: minutes)
+            let timeUntilNotification = lastLoopDate.addingTimeInterval(.minutes(minutes)).timeIntervalSinceNow
+            guard timeUntilNotification >= 0 else { break }
 
             let formatter = DateComponentsFormatter()
             formatter.maximumUnitCount = 1
             formatter.allowedUnits = [.hour, .minute]
             formatter.unitsStyle = .full
 
             let notificationContent = UNMutableNotificationContent()
-            if let failureIntervalString = formatter.string(from: failureInterval)?.localizedLowercase {
+            if let failureIntervalString = formatter.string(from: warningInterval)?.localizedLowercase {
```

Comment:
I think this change is incorrect and believe it should be `timeUntilNotification`. Should the copy be updated to `in at least %@`?

Reply:
Say that lastLoopDate was 15 minutes ago. The "20 minute" loop not looping alert should go off in 5 minutes.  So `timeUntilNotification` should be five minutes.  But the failure string should still indicate that it has been "20 minutes" since the last loop.

Reply:
ah... yup. I agree